tests/Form/GeneralTest.php

```diff
@@ -4,6 +4,7 @@
 
 use Behat\Mink\Exception\DriverException;
 use Behat\Mink\Tests\Driver\TestCase;
+use Behat\Mink\Tests\Driver\Util\FixturesKernel;
```

Comment:
This might not be needed anymore, because `WEB_FIXTURES_DIR` and another constant were moved into the `Behat\Mink\Tests\Driver\TestCase` class.